dagster_university/dagster_essentials/dagster_essentials/completed/lesson_7/definitions.py

```diff
@@ -1,27 +1,13 @@
 import dagster as dg
 
-from dagster_essentials.completed.lesson_7.assets import metrics, trips
-from dagster_essentials.completed.lesson_7.jobs import (
-    trip_update_job,
-    weekly_update_job,
-)
-from dagster_essentials.completed.lesson_7.resources import database_resource
-from dagster_essentials.completed.lesson_7.schedules import (
-    trip_update_schedule,
-    weekly_update_schedule,
-)
-
-trip_assets = dg.load_assets_from_modules([trips])
-metric_assets = dg.load_assets_from_modules([metrics])
-
-all_jobs = [trip_update_job, weekly_update_job]
-all_schedules = [trip_update_schedule, weekly_update_schedule]
-
-defs = dg.Definitions(
-    assets=trip_assets + metric_assets,
-    resources={
-        "database": database_resource,
-    },
-    jobs=all_jobs,
-    schedules=all_schedules,
+import dagster_essentials.completed.lesson_7.defs as defs
+from dagster_essentials.completed.lesson_7.defs.resources import database_resource
+
+defs = dg.Definitions.merge(
```

Comment:
Not sure merging the definitions like this is the correct way to supply resource mapping to `dg.components.load_defs`